swarm/pss/pss_test.go

```diff
@@ -1172,13 +1254,13 @@ func benchmarkSymkeyBruteforceSameaddr(b *testing.B) {
 }
 
 // setup simulated network and connect nodes in circle
-func setupNetwork(numnodes int) (clients []*rpc.Client, err error) {
+func setupNetwork(numnodes int, allowRaw bool) (clients []*rpc.Client, err error) {
```

Comment:
please document `setupNetwork` explaining about `allowRaw`... something along the lines of
```
setupNetwork is creating a simulated network with `numnodes` number of nodes. `allowRaw` is ...
```